src/backend_common.cc

```diff
@@ -606,7 +618,18 @@ CopyBuffer(
   // the src buffer is valid.
```

Comment:
Fix comment here. It says all host to host copy will be blocked but for iGPU we do these memcpy in cuda host functions